pkg/common/utils/resource/pod.go

```diff
@@ -135,6 +137,32 @@ func mergeEnvs(src []corev1.EnvVar, dst []corev1.EnvVar) []corev1.EnvVar {
 	return dst
 }
 
+func NewInitContainer(name string, params []string) corev1.Container {
+	enablePrivileged := true
+	c := corev1.Container{
+		Image:           "alpine:3.16",
```

Comment:
the tag we use latest replaces 3.16, the latest will always exist and use the new feature if the alpine update.